resources/schedules.rb

```diff
@@ -363,6 +383,16 @@ def createSchedule()
     schedule = OpenStudio::Model::ScheduleRuleset.new(@model)
     schedule.setName(@sch_name)
 
+    if not @lower_limit_value.nil? or not @upper_limit_value.nil? or not @numeric_type.nil? or not @unit_type.nil?
```

Comment:
Make this code a method so you don't have identical code two places....

Reply:
Was the HotWaterSchedule class intentionally not touched?

Reply:
Oversight. I will address.